src/System.Resources.Extensions/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+﻿<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
```

Comment:
>4 [](start = 21, length = 1)

do we usually start with 4?

Reply:
yes, that's been the pattern for things in CoreFx, for better or worse, now is not the time to change it.